source/common/http/conn_manager_impl.cc

```diff
@@ -2006,6 +2006,7 @@ void ConnectionManagerImpl::ActiveStreamEncoderFilter::responseDataTooLarge() {
       Http::Utility::sendLocalReply(
           Grpc::Common::hasGrpcContentType(*parent_.request_headers_),
           [&](HeaderMapPtr&& response_headers, bool end_stream) -> void {
+            parent_.chargeStats(*response_headers);
```

Comment:
Thoughts on sending all of the local replies from HCM through a common helper which would track stats so we don't hit this again?